fastapi_sqlalchemy/db.py

```diff
@@ -0,0 +1,21 @@
+"""Database engines management."""
+import sqlalchemy
+
+__ENGINE_REGISTRY = {}
+
+
+def register_engine(engine: sqlalchemy.engine.Engine):
+    """Register given engine wihout any checks."""
+    __ENGINE_REGISTRY[str(engine.url)] = engine
+
+
+def create_engine(url: str, **kwargs) -> sqlalchemy.engine.Engine:
+    """Create an engine and register it."""
+    engine = sqlalchemy.create_engine(url, **kwargs)
+    register_engine(engine)
+    return engine
+
+
+def get_or_create_by_url(url: str) -> sqlalchemy.engine.Engine:
```

Comment:
Add **kwargs here too to pass to create_engine.
We probably also want pool_pre_ping to default to true i.e. check if it exists and force to True if not specificed in kwargs.

Reply:
Added